src/components/SkeletonLoader/_SkeletonLoader.scss

```diff
@@ -1,26 +1,140 @@
-.skeleton-loader:empty {
-  animation: shine 1s infinite;
-  background-image: linear-gradient(
-      100deg,
-      rgba(255, 255, 255, 0),
-      rgba(255, 255, 255, 0.5) 50%,
-      rgba(255, 255, 255, 0) 80%
-    ),
-    linear-gradient(var(--ui-gray-warm-light) 33px, transparent 0),
-    linear-gradient(var(--ui-gray-warm-light) 20px, transparent 0),
-    linear-gradient(var(--ui-gray-warm-light) 20px, transparent 0),
-    linear-gradient(var(--ui-gray-warm-light) 20px, transparent 0),
-    linear-gradient(var(--ui-gray-warm-light) 20px, transparent 0),
-    linear-gradient(var(--ui-gray-warm-light) 20px, transparent 0),
-    linear-gradient(var(--ui-gray-warm-light) 20px, transparent 0);
-  background-position: 0 0, /* highlight */ 0px 0, 0px 40px, 0px 67px, 0px 94px,
-    0px 121px, 0px 148px, 0px 175px;
-  background-repeat: repeat-y;
-  background-size: 50px 230px, /* highlight */ 300px 230px, 250px 230px,
-    250px 230px, 250px 230px, 200px 230px, 200px 230px, 200px 230px;
-  height: 485px; /* change height to see repeat-y behavior */
+@mixin skeleton-highlight {
+  // animation: shine 1s infinite;
+  animation: Pulsate 1000ms cubic-bezier(0.25, -0.5, 1, 0) infinite;
+  background: var(--ui-gray-warm-light);
+  // background-image: linear-gradient(
```

Comment:
If you don't need these, just remove.

Reply:
I am notorious for leaving errant commented code blocks in files.  This has now been cleaned up.  Thanks for keeping me honest.